### PR TITLE
[OSDOCS-4103] Remove outdated limitations re: Kuryr, RHOSP 13, UDP/TCP, and load balancing

### PR DESCRIPTION
**Pull request by maxwelldb**
Version(s): 4.12

Issue: https://issues.redhat.com/browse/OSDOCS-4103

Link to docs preview: https://53911--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-kuryr.html#installation-osp-kuryr-known-limitations_installing-openstack-installer-kuryr

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-osp-kuryr-known-limitations.adoc [hunk 1]
```diff
@@ -35,13 +35,6 @@ services can cause you to run out of resources.
 Deployments of later versions of {rh-openstack} that have the OVN Octavia driver disabled also
```
Comment (dulek) [inline modules/installation-osp-kuryr-known-limitations.adoc:35]: This and paragraph above should probably be reworded too - "versions before 16" basically means 13.

File: modules/installation-osp-kuryr-known-limitations.adoc [hunk 2]
```diff
@@ -50,8 +43,6 @@ and UDP, are not supported.
 
 There are limitations when using Kuryr SDN that depend on your deployment environment.
 
-Because of Octavia's lack of support for the UDP protocol and multiple listeners, if the {rh-openstack} version is earlier than 13.0.13, Kuryr forces pods to use TCP for DNS resolution.
-
 In Go versions 1.12 and earlier, applications that are compiled with CGO support disabled use UDP only. In this case,
```
Comment (dulek) [inline modules/installation-osp-kuryr-known-limitations.adoc:46]: This part until NOTE about musl should be removed too, it's tied to the DNS issue.

Comment (dulek) [reply modules/installation-osp-kuryr-known-limitations.adoc:46]: Then note starting in line 87 should be gone too, 16 always supports UDP.

**Review by dulek — CHANGES_REQUESTED** [review]

File: modules/installation-osp-kuryr-known-limitations.adoc (hunk 1 above)
Comment (maxwelldb) [reply modules/installation-osp-kuryr-known-limitations.adoc:35]: Seems to me that this whole section might be folded into the "general limitations" list. What do you think?

File: modules/installation-osp-kuryr-known-limitations.adoc [hunk 3]
```diff
@@ -35,35 +35,13 @@ services can cause you to run out of resources.
...
-
-To ensure that TCP forcing is allowed, compile applications either with the environment variable `CGO_ENABLED` set to `1`, i.e. `CGO_ENABLED=1`, or ensure that the variable is absent.
-
-In Go versions 1.13 and later, TCP is used automatically if DNS resolution using UDP fails.
-
-[NOTE]
-====
-musl-based containers, including Alpine-based containers, do not support the `use-vc` option.
-====
+* musl-based containers, including Alpine-based containers, do not support the `use-vc` option.
```
Comment (dulek) [inline modules/installation-osp-kuryr-known-limitations.adoc:44]: Seems like this line can be removed too? It's just a workaround for the DNS problem that no longer exists in 16.

**Comment by dulek** [comment]
Just one more comment and besides that LGTM.

**Comment by maxwelldb** [comment]
@dulek Done, I think.

**Comment by maxwelldb** [comment]
@eurijon For review.

**Comment by eurijon** [comment]
lgtm

**Review by mjpytlak — COMMENTED** [review]
@maxwelldb peer review LGTM. Please remember to squash your commits.

**Comment by maxwelldb** [comment]
@mjpytlak TY!

**Pull request merged**